Specification/3._Data_structure_description_language.md

```diff
@@ -132,7 +115,7 @@ Each component is discussed below.
 
 #### Field type
 
-Can be either a primitive data type (primitive data types are defined below) or a nested data structure.
+Fields can be either a primitive data type (primitive data types are defined below) or a nested data structure.
```

Comment:
Very precise :-) Will fix by changing to "The field type can be either a primitive data type ..."